src/views/HomeView.jsx

```diff
@@ -1,16 +1,28 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { UsersList } from "../state/use-users-list";
+
+const HomeView = ({ uname }) => {
+  const { users } = UsersList(uname);
 
-const HomeView = ({ match }) => {
   return (
     <div>
       <ul>
-        <li>
-          <Link to="/marcopeg">Open Marcopeg</Link>
-        </li>
-        <li>
-          <Link to="/luke-skywalker">Open Luke Skywalker</Link>
-        </li>
+        {users.length > 0 ? (
+          users.map(user => (
+            <div key={user.id}>
+              <Link to={`/${user.id}`}>
+                <li>
+                  <h2 className="homeview-user-names">{user.id} </h2>
+                </li>
+              </Link>
+            </div>
+          ))
+        ) : (
+          <div>
```

Comment:
Can you put a `div` inside an `ul`? Or is it better to work this structure a bit differently?